libs/wingsdk/.projenrc.ts

```diff
@@ -54,6 +58,7 @@ const project = new cdk.JsiiProject({
     "vm2",
   ],
   devDeps: [
+    `@cdktf/provider-aws@^12.0.1`, // only for testing Wing plugins
```

Comment:
I tried removing this as a dependency entirely, but then the Wing plugin E2E tests began failing because there was no copy of this dependency they could use. I tried fixing that by adding the dependency in the `examples/plugins` folder but it still couldn't find the dependency. So for now this dependency is just here

Reply:
I think this is perfectly fine